README.md

```diff
@@ -11,11 +27,24 @@ Role Variables
 * `postgres_locale`: Locale used for postgres databases (default to `en_US.UTF-8`).
 * `postgres_default_roles`: List of default user roles (default to `NOSUPERUSER,NOCREATEDB,NOCREATEROLE`).
 
+* `postgres_listen_address`: Listening address of Postgres (default to `localhost`).
```

Comment:
Do we need to support listening on multiple interfaces?

Reply:
Postgres allow `*` to listen on all interfaces and I believe you can set it to something like `192.168.0.191,localhost` to listen on both.

Reply:
Exactly, should we expect them to know they need to pass in a comma separated string, or should we accept an item or list and abstract that for them? I would argue for the latter.

Reply:
I'm kind of torn on this one. Abstracting that is a nice API but I don't think too much abstraction is a good because then you need to learn a new way and can't rely on the pg doc.

Reply:
I guess neither of us know the “user’s use case”! We can evolve this if we find any issues in this approach and recommend users pin their version (I think the users I know of are the site and sirbot)